model/gurps/skill.go

```diff
@@ -973,6 +980,9 @@ func (s *Skill) TechniqueSatisfied(tooltip *xbytes.InsertBuffer, prefix string)
 		return true
 	}
 	e := EntityFromNode(s)
+	if e.SheetSettings.UseSkillTrees {
+		return true
```

Comment:
Do techniques even exist in the skill tree world? I read it when it was released, but haven't looked since, so may just be not remembering them being there.